neurokit2/ecg/ecg_intervalrelated.py

```diff
@@ -2,8 +2,7 @@
 import pandas as pd
 import numpy as np
 
-
-from .ecg_hrv import ecg_hrv
+from ..hrv import hrv_summary
```

Comment:
I wonder if we should call this function simply `hrv()`. In line with a potential future `nk.complexity()` that will compute multiple complexity indices, here it's similar. What do you think?

Reply:
@DominiqueMakowski , I like `hrv()`.